mkosi/__init__.py

```diff
@@ -537,6 +532,25 @@ def install_boot_loader(state: MkosiState) -> None:
                 )
 
 
+def install_skeleton_trees(state: MkosiState, cached: bool) -> None:
+    if not state.config.skeleton_trees or cached:
+        return
+
+    with complete_step("Copying in skeleton file trees…"):
+        for source, target in state.config.skeleton_trees:
+            t = state.root
+            if target:
+                t = state.root / target.relative_to("/")
+
+            t.mkdir(mode=0o755, parents=True, exist_ok=True)
+            if source.is_dir():
+                copy_path(source, t, preserve_owner=False)
+            else:
+                # unpack_archive() groks Paths, but mypy doesn't know this.
+                # Pretend that tree is a str.
```

Comment:
That comment was left behind in 0e8b39a1546f12a99c79ed1dc8cfc72a427d6173. I doesn't make sense now.